client/app/vps/modal/additional-disk/order-disk.controller.js

```diff
@@ -0,0 +1,88 @@
+class OrderAdditionalDiskCtrl {
+    constructor ($translate, $uibModalInstance, CloudMessage, VpsService) {
+        this.$translate = $translate;
+        this.$uibModalInstance = $uibModalInstance;
+        this.CloudMessage = CloudMessage;
+        this.VpsService = VpsService;
+
+        this.capacityArray = [];
+        this.loader = {
+            capacity: false,
+            offer: false,
+            save: false
+        };
+        this.model = {
+            agree: {
+                value: false
+            },
+            capacity: null,
+            date: null,
+            duration: null
+        };
+        this.offer = null;
+
+        this.step = {
```

Comment:
there must be another way? should this.init be a better way, a current step would make a lot more sence and you could use < and >

Reply:
This would be deleted too, I made it quickly before we know if order would be full page or modal to replace wizzard.